##geo fence
geo_fence:
  x_min: -1.0
  x_max: 1.0
  y_min: -1.0
  y_max: 1.0
  z_min: -0.0
  z_max: 1.5
